eng/Versions.props

```diff
@@ -37,6 +37,7 @@
     -->
     <PrivateSourceBuiltSdkVersion>10.0.101-servicing.25569.105</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>10.0.101-servicing.25569.105</PrivateSourceBuiltArtifactsVersion>
+    <DotNet1xxWorkloadManifestVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest100100Version)</DotNet1xxWorkloadManifestVersion>
```

Comment:
nit: add a comment above this line, this helps prevent merge conflicts. might be also a good place to document why we have this dependency